.deepsource.toml

```diff
@@ -0,0 +1,12 @@
+version = 1
```

Comment:
Yes, @dennwc - this is the configuration file for DeepSource, and I would really appreciate it if you could try it out on this repository. I'm interested in hearing your feedback. It's free to use for Open-Source projects.

Reply:
I'm sorry, but why do you need it in the first place?

Our codebase is open, you are free to do whatever you want with the code already, according to the license. You don't need our explicit permission (including this file).

Also, the file itself doesn't contain anything that cannot be done with a heuristic:
- `test_patterns` is common for all Go projects
- `exclude_patterns` can be a heuristic, since it's common to have a `docs` folder
- `name` - can be inferred based on the presence of `go.mod`
- `import_paths` - again, can be inferred from `go.mod` as well

Even if for some reason you don't want to use those heuristics, you could preseed you database with those settings for our repo, since you already did an effort to check it and even file a PR (thank you for that!).

Sorry again if I sound negative, I do want to check the new analyzer in action. But please don't force us to introduce new files that may or may not become stale in the future. We already have issues with maintaining the project :D